### DISCUSSION TITLE
Is it possible to define absorbing boundaries for an unsaturated material in 3D model?

### DISCUSSION THREAD

**Discussion by qizih** [Anura3D/Anura3D_OpenSource#123] [Q&A] [answered]
Hello, everyone!
I'm trying to build a 3D slope model. However, it seems that I can't set soil surface and hydraulic boundary conditions in 3D model unlike 2D model.Can I define an absorbing boundary on the upper surface of the slope to replace the soil surface？
![微信截图_20230731024525](https://github.com/Anura3D/Anura3D_OpenSource/assets/133870124/bd42d0e7-fb8a-4e33-9226-b04d88203e7b)
However, the following error will occur regardless of which hydraulic boundary setting I choose when I follow the above operation. 
![2](https://github.com/Anura3D/Anura3D_OpenSource/assets/133870124/f14779dd-5177-43d1-96ae-cbe22420ee55)
How to set rainfall conditions without applying them to soil surfaces in 3D models？
Best regards,
Qizih

**Comment by FrancescaCeccato** [comment, marked as answer]
I'm sorry but not all our hydraulic BC are yet developed for 3D.
To simulate rainfall you can apply a zero pressure at the top surface (surface saturated).
